libraries/adaptive-expressions/src/builtinFunctions/select.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ExpressionEvaluator } from '../expressionEvaluator';
+import { ReturnType } from '../returnType';
+import { ExpressionType } from '../expressionType';
+import { FunctionUtils } from '../functionUtils';
+
+/**
+ * Operate on each element and return the new collection of transformed elements.
```

Comment:
I don't think this is right is it?  Doesn't it test each element and include those that where the test is true?